browser/components/MarkdownEditor.js

```diff
@@ -142,6 +142,12 @@ class MarkdownEditor extends React.Component {
     this.renderPreview(this.props.value)
   }
 
+  handleKeyDown(e) {
+    if (this.state.status === 'CODE' && e.key == 'Escape') {
```

Comment:
I think the best shortcut is `Ctrl + E`. In order to use it, multiple key is necessary (https://github.com/BoostIO/Boostnote/pull/227)